packages/client-mobile/metro.config.js

```diff
@@ -11,6 +11,8 @@ const config = {
   resolver: {
     // Helps with importing local packages
     nodeModulesPaths: [path.resolve(__dirname, "./node_modules")],
+    unstable_enablePackageExports: true,
+    unstable_conditionNames: ["browser", "require", "react-native"],
```

Comment:
`unstable_enablePackageExports`
- https://reactnative.dev/blog/2023/06/21/package-exports-support

`unstable_conditionNames`
- https://github.com/pmndrs/zustand/discussions/1967
- Only using the `require` hence added it here. If later `browser` or `react-native` will be used by the packages then it should be added to this array